apps/core/app/[locale]/(default)/(faceted)/brand/[slug]/page.tsx

```diff
@@ -117,6 +117,7 @@ export default async function Brand({ params: { slug, locale }, searchParams }:
               prevLabel={tPagination('prev')}
               startCursor={startCursor}
             />
+
```

Comment:
I don't think we want this empty line.

Reply:
sorry for the trouble :D I  run pnpn run lint on core package